packages/remix-deno/server.ts

```diff
@@ -94,11 +94,7 @@ export function createRequestHandlerWithStaticFiles<Context = unknown>({
   return async (request: Request) => {
     try {
       return await serveStaticFiles(request, staticFiles);
-    } catch (e: any) {
-      if (e.code !== "EISDIR" && e.code !== "ENOENT") {
-        throw e;
-      }
-    }
+    } catch (e) {}
```

Comment:
Let's add a comment inside the `catch` that explains why we're not handling this.